COMMITTERS.rst

```diff
@@ -15,14 +15,39 @@
     specific language governing permissions and limitations
     under the License.
 
+Committers and PMC members
+==========================
+
+This document assumes that you know how Airflow's community work, but you would like to learn more about the rules by which we add new members.
+
+Before reading this document, you should be familiar with `Contributors' guide <contributing-docs/README.rst>`__.
+This document assumes that you are a bit familiar how Airflow's community work, but you would like to learn more
+about the rules by which we add new members.
```

Comment:
Duplicate text, I assume this is a leftover?
```suggestion
Before reading this document, you should be familiar with `Contributors' guide <contributing-docs/README.rst>`__.
This document assumes that you are a bit familiar how Airflow's community work, but you would like to learn more
about the rules by which we add new members.
```

Reply:
Yeah. Bad merge :).